### DISCUSSION TITLE
Adding a local backup instance

### DISCUSSION THREAD

**Discussion by alejandro-alzate** [libre-tube/LibreTube#4926] [Ideas]
What about an opt in option to enable a local instance that automatically kicks in when external instances are down or slow

**Comment by ghost** [comment]
> What about an opt in option to enable a local instance that automatically kicks in when external instances are down or slow

You can bypass instance video proxying and connect directly to youtube servers.

Settings -> Instances -> Proxy -> Disable Piped Proxy

Local private instances wouldn't serve a purpose as it is pretty much just like connecting directly to youtube servers if you are the only one using it. 

The point of the instances and proxying requests is to _scramble_ the data that goes to (~~g~~)oogle so that they are unaware of who is searching what. They only know the server is requesting all the videos but they do not know which individuals use that instance, making this data invaluable to them.

_Any questions feel free to ask :)_
_I'm not an expert but I'll answer to my best understanding, feel free to correct me._

**Comment by alejandro-alzate** [comment]
Oh yes you have to excuse me i just found that option yesterday, thanks for the time though.

**Comment by alejandro-alzate** [comment]
may i change the goal of this discusion the feature is there (my bad, i was unaware), what about a prompt saying the downsides and benefits when it gets stuck for too long and too many timrs that tells the user either to change instansce or try out disabling the proxying, my failure was that i found the option because i got in the wrong menu trying to backup the config before updating to the hotfix update that dealt with that pesky T in the timestamp.
